versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
+
+The serialization of the request body should be specified by the `consumes` attribute.
```

Comment:
I'm not sure if/how the `consumes` attribute actually specifies how the request body is being serialized. Do you mean something like

>  If we have an `application/json` or `application/...+json` media type, it is serialized as JSON, if we have an `application/xml` or `application/...+xml` media type, it is serialized as XML?

If so, it should actually be written out.